docs/guides/intersphinx.rst

```diff
@@ -0,0 +1,163 @@
+Link to Other Projects' Documentation With Intersphinx
+======================================================
+
+You may be familiar with using the :ref:`:ref: role <sphinx:ref-role>` to link to any location of your docs.
+It helps you to keep all links within your docs up to date and warns you if a reference target moves or changes
+so you can ensure that your docs don't have broken cross-references.
+
+Some times you may need to link to a location of another documentation project.
+We could just link to where the documentation is hosted.
+
```

Comment:
I would remove the small paragraph above and combine it into the paragraph below. Something like:

```
Sometimes you may need to link to a specific section of another project's documentation.
While you could just hyperlink directly, there is a better way.
```